source/process/community-overview.rst

```diff
@@ -44,31 +46,31 @@ Audiences
 1. Integration builders (Owner: Jason)
 
   - They create extensions to Mattermost to third-party integrations and plug-ins.
-  - Some might build drivers to more easily integrate Mattermost into different programming languages
-  - Examples: Popular third-party software (e.g. Jira, Jenkins), Slack-compatible integrations (e.g. Slackbot), Custom Mattermost integrations (e.g. Matterbridge, Mattermail), Plug-ins (to come)
+  - Some might build drivers to more easily integrate Mattermost into different programming languages.
+  - Examples: Popular third-party software (e.g. Jira, Jenkins), Slack-compatible integrations (e.g. Slackbot), Custom Mattermost integrations (e.g. Matterbridge, Mattermail), Plug-ins (e.g. Zoom).
 
 2. Code contributors (Owner: Eric)
 
-  - They contribute features, small improvements and bug fixes to server, webapp, mobile and desktop app repos
+  - They contribute features, small improvements and bug fixes to server, webapp, mobile and desktop app repos.
 
-3. End users (Owner: Lindy)
+3. End users (Owner: Paul)
 
-  - They ask for help troubleshooting issues, report bugs, share UX feedback and propose new features
-  - Examples: forum.mattermost.org, GitHub repos, Uservoice
+  - They ask for help troubleshooting issues, report bugs, share UX feedback and propose new features.
+  - Examples: forum.mattermost.org, GitHub repos, Uservoice.
 
 4. Localization (Owner: Jason)
 
-  - They help translate and maintain languages in Pootle translation server
+  - They help translate and maintain languages in Pootle translation server.
 
-5. Orchestration solution maintainers (Owner: TBD)
+5. Orchestration solution maintainers (Owner: Lindy)
```

Comment:
@amyblais Or is the owner yourself since you're contacting the orchestration maintainers about new releases?